Ai

MarcelCerdan opened the pull request:
Added a local game menu in wich you can choose to play against an AI or another player locally.
Added the AI, it will need a little more training i think.
I erased the part where the AI is trained at the end of a game.
Nothing changes in the game itself.
I couldn't try the last commit on my computer so feel free to try everything event the multiplayer to be sure it didn't broke anything but it should be good.

boulatrousse reviewed (changes requested):
- AI does not work and my game crashed with this error : 
![image](https://github.com/user-attachments/assets/509b0ae8-9a1d-4d76-95f7-534fd1ecb1cd)

- Please resolve conflicts before request reviews : 
![image](https://github.com/user-attachments/assets/49c0ada1-70b6-44ee-83b0-b6ece247243f)

- In `users/models.py`, don't push migrations that will be change in the future. Please set `email_is_verified` to False.

MarcelCerdan commented:
I fixed the problems found by @boulatrousse.
I also added an intermediate mode with the easy mode.
The subject says that the AI should not receive the game state in live so i added a timer in the easy mode. It really reduces the AI level so there I deleted this in the intermediate mode.

maujogue reviewed (approved):
IA is not very good on my computer because the ball is going faster, I could not get more than a few touch from it.
